modules/java_api/cpp/infer_request.cpp

```diff
@@ -16,6 +16,22 @@ JNIEXPORT void JNICALL Java_org_intel_openvino_InferRequest_Infer(JNIEnv *env, j
         infer_request->infer();)
 }
 
+JNIEXPORT void JNICALL Java_org_intel_openvino_InferRequest_StartAsync(JNIEnv *env, jobject obj, jlong addr)
```

Comment:
Is it safe to add more methods without tests?

Reply:
Good point! Will add some tests

Reply:
Tests added. Please review